ui/src/tool-bar/tool-bar.tsx

```diff
@@ -403,22 +427,38 @@ export const ToolBar: React.FC<ToolbarParams> = ({
                 )}
                 {(layoutContext.designId || selectingWorkspace) && (
                     <React.Fragment>
-                        {/* TODO Add functionality once design projects have a data model */}
                         <Dropdown
                             inputRef={selectWorkspaceDropdownRef}
                             placeholder={t('tool-bar.choose-workspace')}
-                            openOverride={selectingWorkspace ? true : undefined}
+                            openOverride={selectingWorkspace || undefined}
```

Comment:
Näyttää ehkä oudolta, mutta käytännössä tämä pakottaa parametrin arvoksi `false`:n sijasta `undefined`:in